src/core/map/hooks/useMapPositionTracker.ts

```
@@ -0,0 +1,116 @@
+import { useEffect, useRef, useCallback, useMemo } from 'react';
+import { throttle } from '@github/mini-throttle';
+import { geographicToPageCoords } from '../utils/maplibreCoordinateUtils';
+import { isValidLngLatArray } from '../utils/coordinateValidation';
+import type { Map } from 'maplibre-gl';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)